code/modules/modular_computers/file_system/programs/system/file_browser.dm

```diff
@@ -153,6 +150,23 @@
 			return 1
 		var/datum/computer_file/C = F.clone(0)
 		HDD.store_file(C)
+	if(href_list["PRG_encrypt"])
+		. = 1
+		var/obj/item/weapon/computer_hardware/hard_drive/HDD = computer.hard_drive
+		var/datum/computer_file/F = HDD.find_file_by_name(href_list["PRG_openfile"])
```

Comment:
shouldnt that be ?
`var/datum/computer_file/F = HDD.find_file_by_name(href_list["PRG_encrypt"])`
-> Same for the decrypt option below